babel.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
         // TODO: change this to `automatic` when using the new runtime
         runtime: 'classic',
         keepPropTypes: true,
+        enableLooseMode: process.env.CYPRESS_CI === 'true' ? false : true,
```

Comment:
Cypress hat its own webpack config. Even when enabling all three plugins to be loose Cypress seems to overwrite it as it doesn't want to run in loose mode. We can however disable loose mode. It's even better for debugging but slower for performance. There won't be a browser compatibility difference.

Reply:
I would probably make this opt out.

```suggestion
        disableLooseMode: process.env.CYPRESS_CI === 'true',
```